src/types/streaming_server/device_info.rs

```diff
@@ -0,0 +1,7 @@
+use serde::{Deserialize, Serialize};
+
+#[derive(Clone, PartialEq, Serialize, Deserialize, Debug)]
+#[serde(rename_all = "camelCase")]
+pub struct DeviceInfo {
+    pub available_hardware_accelerations: Vec<String>,
```

Comment:
- [ ] FIXME: the server can actually return `false` when there are no available HW accelerations